alacritty/src/config/bindings.rs

```diff
@@ -735,15 +733,15 @@ impl<'a> Deserialize<'a> for BindingKey {
                 } else {
                     // Translate legacy winit codes into their modern counterparts.
                     match keycode.as_str() {
-                        "Up" => (Key::Named(ArrowUp), KeyLocation::Standard),
```

Comment:
Couldn't this just be `NamedKey::ArrowUp`? I'd much prefer that since it's much easier to actually understand for someone writing new bindings.

Reply:
it's conversion code and not bindings. Not sure what to do here?